src/main/java/com/google/sps/servlets/TasksServlet.java

```diff
@@ -52,6 +61,70 @@ public TasksServlet(
     this.tasksClientFactory = tasksClientFactory;
   }
 
+  private List<Task> getTasks(TasksClient tasksClient) throws IOException {
+    List<TaskList> taskLists = tasksClient.listTaskLists();
+    List<Task> tasks = new ArrayList<>();
+    for (TaskList taskList : taskLists) {
+      tasks.addAll(tasksClient.listTasks(taskList));
+    }
+    return tasks;
+  }
+
+  private List<String> getTaskListTitles(List<TaskList> taskLists) throws IOException {
+    return taskLists.stream().map(taskList -> taskList.getTitle()).collect(Collectors.toList());
+  }
+
+  private int getTasksToComplete(List<Task> tasks) {
+    List<Task> tasksCompletedToday =
```

Comment:
Will this return all tasks that have been completed, or just the ones completed today? If there is a restriction on the input to make the latter true, document this in a comment. This might actually be the variable name that's confusing me here, since the method is called `getTasksToComplete` but the variable is called `tasksCompletedToday`

Reply:
I missed that variable, must have copied the code from another method and forgot to change the variable name. Thanks for pointing it out!

Reply:
`RESOLVED`